PowerValidatedSolutions.psm1

```diff
@@ -59392,6 +59398,87 @@ Function Test-NtpServer {
 }
 Export-ModuleMember -Function Test-NtpServer
 
+Function Test-PrereqDnsEntries {
+    <#
+    .SYNOPSIS
+    Checks the DNS resolution of a DNS entry or entries on a DNS server or servers.
+
+    .DESCRIPTION
+    The Test-PrereqDnsEntries cmdlet checks for the resolution of a DNS record or multiple DNS records and tests the forward and reverse lookups in a DNS server or multiple DNS servers.
+
+    .EXAMPLE
+    Test-PrereqDnsEntries -dnsEntries sfo-vcf01.sfo.rainpole.io, 192.168.11.41 -dnsServers rpl-ad01.rainpole.io
+    This example will validate the forward and reverse details of the two DNS entries and if one exists in the DNS Server and return the cooresponding value.
+
+    .PARAMETER dnsEntries
+    The fully qualified domain name (FQDN) or IP address of the DNS entry or entries to check, separated by commas if checking more that one.
+
+    .PARAMETER dnsServers
+    The fully qualified domain name (FQDN) or IP address of the DNS server or servers to check for the DNS entry or entries in separated by commas if checking more that one.
```

Comment:
```suggestion
    The fully qualified domain name (FQDN) or IP address of the DNS server or servers to check for the DNS entry or entries in separated by commas if checking more than one.
```

Reply:
Updated In https://github.com/vmware/power-validated-solutions-for-cloud-foundation/commit/839b36abdacf09ae8380c818b8ebaab40f61f34a